.gitignore

```diff
@@ -0,0 +1,70 @@
+# OSX
+.DS_Store
```

Comment:
This kind of files belong to your own global .gitignore file (.DS_Store, Thumb.db, Desktop.ini, .idea/ and so on)